Resources/Prototypes/Entities/Objects/Weapons/Guns/turrets.yml

```
@@ -128,6 +128,48 @@
 
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponTurretSAN
+  suffix: Sol Alliance Navy
+  description: A turret designed to provide protection for Sol Marines during combat drops. It menaces targets with a twin-linked C-20r, although its cheap fire controller lends to poor accuracy.
+  components:
+    - type: NpcFactionMember
+      factions:
+        - SAN
+    - type: Gun
+      fireRate: 10 # Guaranteed to spook the shit out of people.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)